app/models/application_record.rb

```diff
@@ -1,3 +1,6 @@
+# frozen_string_literal: true
+
+# Application record class
 class ApplicationRecord < ActiveRecord::Base
```

Comment:
app/jobs/application_job and this /models/application_record file could be deleted for now. Not going to be used.